### DISCUSSION TITLE
I am using quasar framework for making my website responsive

### DISCUSSION THREAD

**Discussion by Alam487** [quasarframework/quasar#10119] [CLI - BEX mode] [answered]
Hi, Here I am using quasar framework for making my ecommerce website  responsive like in "mobile and tablet". Here I am using 

```
<head>
    <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900|Material+Icons" rel="stylesheet" type="text/css">
    <link href="https://cdn.jsdelivr.net/npm/quasar@2.0.3/dist/quasar.prod.css" rel="stylesheet" type="text/css">
  </head>
```

In all the divs I am using like..
```
<div id = "q-quasar"></div>
```
In body section I am using this 

```
<script src="https://cdn.jsdelivr.net/npm/vue@3/dist/vue.global.prod.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/quasar@2.0.3/dist/quasar.umd.prod.js"></script>
```

And also I have used this 

```
<script>
      const app = Vue.createApp({
        setup () {
          return {}
        }
      })

      app.use(Quasar)
      app.mount('#q-app')
    </script>
```

But this is not affecting to my site. So please tell me where I am going wrong. In mobile responsive it is not working

**Comment by sasgosh** [comment]
Hello,

You have div id="q-quasar" and app.mount('#q-app')... it'not same id. 
Everything else seems ok to me.

**Comment by hawkeye64** [comment, marked as answer]
A couple of things:
1) Change `quasar@2.0.3` to just `quasar@2` so you always get the latest bug fixes for Quasar v2 (just the same way you have `vue@3`)

2) `const app = Vue.createApp()`
If you have no code here, then don't add and return an empty `setup()` function

3) `app.use(Quasar, { config: {} })`
You need to fill in the config sections: https://quasar.dev/start/umd#quasar-config-object

4) Using this: `app.mount('#q-app')` is correct, but your html is not. The correct way is: `<div id="q-app" ...`
